CHANGES.md

```diff
@@ -42,6 +42,7 @@ replace with next unreleased version
 
 ### NEW FEATURES
 
+- [DDF-2467](https://codice.atlassian.net/browse/DDF-2467) Initial support for Solr Cloud
```

Comment:
That is what I was going for with initial.  We do not have a standard defined way to present features as Beta currently.  Should we include Beta in the bundle name?  It should be added to the documentation at least.  Not sure about the release notes.